_drafts/SC1-2-2-video-captions.md

```diff
@@ -0,0 +1,88 @@
+---
+name: video captions
+description: |
+  Captions are available for audio information in non-streaming `video` elements.
+
+success_criterion:
+- 1.2.2 # Captions (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output][]
+- [Visual output][]
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] `video` element [visible on the page][] where the video contains [audio][].
```

Comment:
This SC applies to all prerecorded content (i.e., non-live content). From what I can understand from the definition of non-streaming and HTMLMediaElement, non-streaming is not equivalent to non-live. Shouldn't the rule should apply to non-live video elements?

Additionally, if the audio information is available on the page it is not required to be captioned. Therefore, this rule should not be applicable when the information is available on the page.

Reply:
Should there be an exception for "when the media is a media alternative for text and is clearly labeled as such"?

Reply:
We looked at that, and the problem of "live" is that it's impossible to determine. A video can be live and you can never know about it - or it could indicate that it's live when it actually isn't. The thing we settled on is that any live video has to be streaming. If you can skip to the end of the video, you know it isn't live. So that's what we're testing. This excludes videos that are streaming but aren't live, but I don't know of a way to test that reliably.

This is a matter of us scoping down the rule to ensure it can be tested consistently.

Reply:
@annethyme We'll add a rule for media alternatives - that should address this.